creativecloud/scripts/utils.js

```diff
@@ -22,18 +22,15 @@
 export const [setLibs, getLibs] = (() => {
   let libs;
   return [
-    (prodLibs) => {
-      const { hostname } = window.location;
-      if (!hostname.includes('hlx.page')
-        && !hostname.includes('hlx.live')
-        && !hostname.includes('localhost')) {
-        libs = prodLibs;
-        return libs;
-      }
-      const branch = new URLSearchParams(window.location.search).get('milolibs') || 'main';
-      if (branch === 'local') return 'http://localhost:6456/libs';
-      if (branch.indexOf('--') > -1) return `https://${branch}.hlx.page/libs`;
-      return `https://${branch}--milo--adobecom.hlx.live/libs`;
+    (prodLibs, location) => {
```

Comment:
We don't need to pass location as 
{ hostname, search } = window.location;
will get the correct value
This will also change the standard definition of getLibs

Reply:
The definition of getLibs does change but I don't think that impacts anything in our code base and any code update will be needed. I have updated the function as per the update done in milo-college project here https://github.com/adobecom/milo-college/blob/main/scripts/utils.js

I agree this change is not required in our use case right now where libs is returned undefined in certain conditions. But I have updated it as per milo code so that we are in sync with latest milo changes.